.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     # Pull docker image from docker hub
     # XTERM used for better catkin_make output
     docker:
-      - image: usdotfhwastol/carma-base-ci:2.9.0
+      - image: usdotfhwastol/autoware.ai:3.1.0-Test
```

Comment:
I haven't been following the release process much, but I believe you want this tag to reference 3.1.0 not -Test